controllers/ie_controller_test.go

```diff
@@ -0,0 +1,177 @@
+package controllers
```

Comment:
Unless the testify package needs this file to be in the same package as the controller it is testing, I would prefer it to follow the idiom of being in `package controllers_test` so that we can make sure the test isn't getting too involved with the implementation of a controller.